tests/accelerators/test_accelerator_connector.py

```diff
@@ -119,7 +120,7 @@ def on_fit_start(self, trainer, pl_module):
         trainer.fit(model)
 
 
-@pytest.mark.skipif(not torch.cuda.is_available(), reason="test requires GPU")
+@SkipIf(min_gpus=1)
```

Comment:
This name SkipIf, is it already decided?
Because the way you read it is not natural at all, and it expresses quite the opposite of what it should :) 
I think everyone who will see this the first time will scratch their head a few times 😄

Reply:
Perhaps RunIf would make more sense?

Reply:
ok, `RunIf` sounds good :] renamed

Reply:
thanks! 
btw any particular reason why it is capitalized / camel case?
no objection, just an observation

Reply:
it is Class, which in python is camelcase

Reply:
moreover in this min, the RunIf makes sense but how about later skip if Windows is true, then it reverse back...